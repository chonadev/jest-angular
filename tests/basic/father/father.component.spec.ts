import { ComponentFixture, TestBed } from '@angular/core/testing';

import { FatherComponent } from '../../../src/app/basic/father/father.component';

describe('FatherComponent', () => {
  let component: FatherComponent;
  let fixture: ComponentFixture<FatherComponent>;
  let compiled: HTMLElement;
  beforeEach(async () => {
    await TestBed.configureTestingModule({
      declarations: [ FatherComponent ]
    })
    .compileComponents();

    fixture = TestBed.createComponent(FatherComponent);
    component = fixture.componentInstance;
    fixture.detectChanges();
    compiled = fixture.nativeElement;
  });

  test('should create', () => {
    expect(component).toBeTruthy();
  });

  test('debe hacer match con el snapshot', () => {
    expect( compiled ).toMatchSnapshot();
  });

});
