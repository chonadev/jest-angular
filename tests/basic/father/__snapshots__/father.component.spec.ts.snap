// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FatherComponent debe hacer match con el snapshot 1`] = `
<div
  id="root1"
>
  <h1>
    Padre
  </h1>
  <button>
    Cargar cliente
  </button>
  <br />
  <code>
     

  </code>
  <hr />
  <app-father-son />
</div>
`;
